charts/pega/templates/_pega-service.tpl

```diff
@@ -22,9 +22,12 @@ metadata:
     traefik.ingress.kubernetes.io/max-conn-amount: '10'
     # Manually set the cookie name for sticky sessions
     traefik.ingress.kubernetes.io/session-cookie-name: UNIQUE-PEGA-COOKIE-NAME
+    # Sets serversTreansport that has config in order to verify rootCA
+    traefik.ingress.kubernetes.io/service.serverstransport: {{ .root.Release.Namespace }}-{{ .name }}-servers-transport@kubernetescrd
   {{- else if (eq .root.Values.global.provider "gke") }}
   annotations:
     cloud.google.com/neg: '{"ingress": true}'
+    cloud.google.com/app-protocols: '{"https":"HTTPS","http":"HTTP"}'
     {{ if (semverCompare "< 1.22.0-0" (trimPrefix "v" .root.Capabilities.KubeVersion.GitVersion)) }}beta.{{ end -}}cloud.google.com/backend-config: '{"ports": {"{{ .node.service.port }}": "{{ .name }}"}}'
```

Comment:
Can `app-protocals` set always to http and https? Same applies to `traefik.ingress.kubernetes.io/service.serverstransport` annotation. Can these be safely set even though `tls.enabled` is false

Reply:
This annotation wont cause any harm, it can be safely set like others.
app-protocols can be set, it wont case any issue